lib/iex/lib/iex/parser.ex

```diff
@@ -0,0 +1,36 @@
+defmodule IEx.Parser do
+  @moduledoc """
+  The base implementation of the parser we use before evaluating iex code
+
+  This parser can be replaced by setting the `:parser` attribute of the iex
+  config
+  """
+
+  # The first two clauses provide support for the break-trigger allowing to
+  # break out from a pending incomplete expression. See
+  # https://github.com/elixir-lang/elixir/issues/1089 for discussion.
+  @break_trigger '#iex:break\n'
```

Comment:
Note that this needs to be a string:

```suggestion
  @break_trigger "#iex:break\n"
```